cypress/components/alert/alert.cy.tsx

```diff
@@ -101,13 +101,13 @@ context("Testing Alert component", () => {
     );
 
     it.each([
```

Comment:
This should help with simplifying the type declaration a little bit, by instead writing the input as an array containing `[string, number]` tuples: 

```tsx
it.each([
      [SIZE.EXTRASMALL, 300],
      [SIZE.SMALL, 380],
      [SIZE.MEDIUMSMALL, 540],
      [SIZE.MEDIUM, 750],
      [SIZE.MEDIUMLARGE, 850],
      [SIZE.LARGE, 960],
      [SIZE.EXTRALARGE, 1080],
    ] as [string, number][])(
```